tools/scripts/build_projects.py

```diff
@@ -192,11 +192,11 @@ def build(self):
 				new_hdf = True
 		
 		if new_hdf:
-			err = run_cmd(cmd + ' clean_all')
+			err = run_cmd(cmd + ' reset')
 			if err != 0:
 				return err
 
-		err = run_cmd(cmd + ' update_srcs')
+		err = run_cmd(cmd + ' update')
 		if err != 0:
 			return err
 		err = run_cmd(cmd + ' -j%d re' % (multiprocessing.cpu_count() - 1))
```

Comment:
`make re` was removed with d71c363baa61b574042e58f3a037ef32d350c02e

Reply:
yes, true, we need to make sure all the changes in this PR don't break our CI builds

Reply:
Done